libvast/CMakeLists.txt

```diff
@@ -578,22 +582,32 @@ endif ()
 
 # Install libvast in PREFIX/lib and headers in PREFIX/include/vast.
 install(
-  TARGETS libvast libvast_native_plugins
+  TARGETS libvast
+  EXPORT VASTTargets
+  ARCHIVE DESTINATION "${CMAKE_INSTALL_LIBDIR}" COMPONENT Development
+  LIBRARY DESTINATION "${CMAKE_INSTALL_LIBDIR}" COMPONENT Runtime
+  RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}" COMPONENT Runtime)
+
+install(
+  TARGETS libvast_native_plugins
```

Comment:
Can you remind me again why we are even installing this?

Reply:
The plugin unit test binaries need to be able to link against it.

Reply:
I'll add a comment for this.